RELEASE_NOTES.md

```diff
@@ -17,6 +17,8 @@
   microgrid metadata. The returned value is an instance of the message
   `Metadata`.
 
+  Note that this RPC has been renamed to `GetMicrogridMetadata` in [this PR](https://github.com/frequenz-floss/frequenz-api-microgrid/pull/43).
```

Comment:
If this is a new feature, I would just update the previous release notes item to use the final name.

Reply:
Done 🚀